src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -5911,6 +5920,49 @@ void FillWeatherPredefinedEntries(EnergyPlusData &state)
                     PreDefTableEntry(state, state.dataOutRptPredefined->pdchWthrVal, "Minimum Dew Point Occurs on", "not found");
                 }
             } break;
+            case StatLineType::MonthlyPrec: { //   - Monthly precipitation mm
```

Comment:
It appears that this section is adding rows to an existing output table? If so, then this PR should have the "OutputChange" label and the changes should be described in \src\Transition\OutputRulesFiles\OutputChanges9-6-0-to-22-1-0.md

Reply:
Thanks, the PR is now labeled with OutputChange as well.